test/e2e_node/util.go

```diff
@@ -365,3 +367,26 @@ func runCommand(cmd ...string) (string, error) {
 	}
 	return string(output), nil
 }
+
+// getCRIClient connects CRI and returns CRI runtime service clients and image service client.
+func getCRIClient() (internalapi.RuntimeService, internalapi.ImageManagerService, error) {
+	// connection timeout for CRI service connection
+	const connectionTimeout = 2 * time.Minute
+	runtimeEndpoint := framework.TestContext.ContainerRuntimeEndpoint
+	r, err := remote.NewRemoteRuntimeService(runtimeEndpoint, connectionTimeout)
+	if err != nil {
+		return nil, nil, err
+	}
+	imageManagerEndpoint := runtimeEndpoint
+	if framework.TestContext.ImageServiceEndpoint != "" {
+		//ImageServiceEndpoint is the same as ContainerRuntimeEndpoint if not
+		//explicitly specified
+		//https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L517
```

Comment:
Change the link to point to a versioned code, or just remove the link completely.

Reply:
WIll remove it.

Reply:
Done.